decidim-core/app/models/decidim/user_role_config/base.rb

```diff
@@ -7,8 +7,20 @@ def initialize(user_role)
         @user_role = user_role
       end
 
-      def component_is_whitelisted?(_manifest)
-        true
+      # Checks whether the given manifest name is accepted for this role.
```

Comment:
Can you explain here what does it means/imply when a role accepts/whitelists a manifest name?

Also, the term "white/black list" is being replaced from some of the leading Ruby gems for its racist implications. We don't want to change it now here, but take it into account for future core ;)

Reply:
The issue is that, without this, valuators could list all components in a space, even if they couldn't perform any action to them. Product asked to hide the list of inaccessible components, a concept that didn't exist until now because the valuator role is the first one that is limited to a specific component (proposals, in this case).

I tried exploring the permissions system, but I couldn't find an easy way to get the results I wanted. I'm having second thoughts, though, because the new system I added seems to be conceptually interfering with Permissions, and to be fair the system is complex enough to be adding new parallel systems.

I'll rethink this approach and ping you again for reviews, sorry!

Reply:
The idea behind this change is to let user roles specify what components they can access. This is a new concept that wasn't previously implemented, as all roles could access all components until the valuator one was added.

I'll replace `whitelisted` for `accessible`, to keep intentions clearer.